From fb2a063e48d311884432ad789af002df89f66239 Mon Sep 17 00:00:00 2001
From: Dan Petrisko <petrisko@cs.washington.edu>
Date: Wed, 6 May 2020 15:29:00 -0700
Subject: [PATCH] Fix

---
 include/verilated.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/verilated.cpp b/include/verilated.cpp
index c242b5ab..4bef749b 100644
--- a/include/verilated.cpp
+++ b/include/verilated.cpp
@@ -33,6 +33,7 @@
 #include <algorithm>
 #include <cctype>
 #include <cerrno>
+#include <sstream>
 #include <sys/stat.h>  // mkdir
 
 #if defined(_WIN32) || defined(__MINGW32__)
-- 
2.16.5

